runtime/near-vm/lib/api/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "wasmer-near"
-version = "2.4.0"
+version = "0.0.0"
 description = "High-performance WebAssembly runtime"
 categories = ["wasm"]
 keywords = ["wasm", "webassembly", "runtime", "vm"]
-authors = ["Wasmer Engineering Team <engineering@wasmer.io>"]
+authors = ["Near Inc <hello@nearprotocol.com>"]
```

Comment:
I don’t believe we should be removing the Wasmer Engineering Team attribution – MIT requires it. Adding ourselves is :+1: though.

While at that, the license files should be adjusted to include ourselves too, without changing the wasmer attribution.